VideoAPI/VideoApi.DAL/Exceptions/RoomParticipantNotFoundException.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace VideoApi.DAL.Exceptions
+{
+#pragma warning disable S3925 // "ISerializable" should be implemented correctly
+
+    public class RoomParticipantNotFoundException : Exception
```

Comment:
Could you implement the exception like so [Notification API exceptions](https://github.com/hmcts/vh-notification-api/blob/master/NotificationApi/NotificationApi.DAL/Exceptions/NotificationNotFoundException.cs)

Reply:
:) our now Gold Standard project

Reply:
implemented, done